src/stream/src/executor/aggregation/mod.rs

```diff
@@ -60,8 +60,8 @@ pub fn generate_agg_schema(
 }
 
 pub async fn agg_call_filter_res(
-    ctx: &ActorContextRef,
-    identity: &str,
+    _ctx: &ActorContextRef,
+    _identity: &str,
```

Comment:
What about just removing these arguments? Also elsewhere.

Reply:
Okay. I'll remove all function parameters but keep the existing ones in the executor body for potential future use. 😄